book/src/simple-testnet.md

```diff
@@ -1,47 +1,44 @@
 # Simple Local Testnet
 
-With a functional [development environment](./setup.md), starting a local multi-node
-testnet is easy:
+First, [install Lighthouse](./installation.md).
 
-1. Start the first node: `$ lighthouse bn testnet -f recent 8`
-1. Start a validator client: `$ lighthouse bn testnet -b insecure 0 8`
-1. Start more nodes with `$ lighthouse bn -b 10 testnet -f bootstrap
-   http://localhost:5052`
-   - Increment the `-b` value by `10` for each additional node.
+Then, get the current unix time in seconds; you can use
+[epochconverter.com](https://www.epochconverter.com/) or `$ date +%s`. It
+should look like this `1576803034` and you should use it wherever we put
+`<time>`.
 
-## Detailed Instructions
-
-First, setup a Lighthouse development environment and navigate to the
-`target/release` directory (this is where the binaries are located).
+> If you choose a time that's more than several minutes in the past the
+> validator client will refuse to produce blocks. We will loosen this
+> restriction in the future, the issue is tracked
+> [here](https://github.com/sigp/lighthouse/issues/714).
 
 ## Starting a beacon node
 
-Start a new node (creating a fresh database and configuration in `$HOME/.lighthouse`), using:
+Start a new node with:
 
 ```bash
-$ lighthouse bn testnet -f recent 8
+$ lighthouse bn --http testnet -r quick 8 <time>
 ```
 
 > Notes:
 >
-> - The `-f` flag ignores any existing database or configuration, backing them
->   up before re-initializing.
+> - The `--http` flag starts the API so the validator can produce blocks.
+> - The `-r` flag creates a random data directory to avoid clashes with other
+>    nodes.
 > - `8` is number of validators with deposits in the genesis state.
-> - See `$ lighthouse bn testnet recent --help` for more configuration options,
+> - See `$ lighthouse bn testnet minimal --help` for more configuration options,
```

Comment:
I guess this should be just `$ lighthouse bn testnet --help`

Reply:
Good spotting, thanks!